### ISSUE TITLE
[react] Tour tip is missing a section role on the outer element

### ISSUE THREAD

**Issue by aaron-nance** [eBay/evo-web#108]
For accessibility and to be in parity with what Skin dictates, our tour tip should have a section role on the outer element. This will help screen readers identify it.
